wurst/network/Metadata.wurst

```diff
@@ -1,52 +0,0 @@
-package Metadata
-import GamecacheKeys
-
-public tuple metadata(int ints, int reals, int booleans, int asciiInts, int syncs)
-
-public constant EMPTY_META = metadata(0, 0, 0, 0, 0)
-
-/**
-	Simple wrapper class for Network-related metadata about the transmission.
-	Necessary to let other clients know about the amount of data being sent.
-**/
-public class MetadataStore
```

Comment:
Delete instead of deprecate?

Reply:
Network doesn't work since the patches. This is only used by Network.